src/transport/SecureSession.h

```diff
@@ -66,7 +71,9 @@ class SecureSession
     PeerAddress & GetPeerAddress() { return mPeerAddress; }
     void SetPeerAddress(const PeerAddress & address) { mPeerAddress = address; }
 
+    SessionType GetSessionType() const { return mSessionType; }
```

Comment:
So maybe that enum should live here and be `SecureSession::Type` or something?

Reply:
done